spec.md

```diff
@@ -615,6 +621,24 @@ message PublishVolumeInfo {
   // possible.
   map<string, string> values = 1;
 }
+
+message Principal {
+  // describes attributes for defining the principal identity of
+  // a volume binding for a container.
+
+  // This field is OPTIONAL. If specified it contains the CO namespace
+  // identity in which the the following identity information (user /
+  // group) exists.
+  string namespace = 1;
+  // This field is OPTIONAL. If specified it contains the CO user
+  // identity holding ownership of a volume, similar to the user_id
+  // in a POSIX file system.
+  string user = 2;
+  // This field is OPTIONAL. If specified it contains the CO group
+  // identities holding ownership of a volume, similar to the group_id
+  // in a POSIX file system.
+  repeated string group = 3;
+}
```

Comment:
@jdef : Thanks for the hierarchy pointers, that helped!

In essence the question if containers in a hierarchy use the same principal or not is up to the CO as the CO provides the principal information for publishing a given mount. If the CO enforces using the same principal for different mounts in a set of containers, then all containers in that set will use the same identity to access the volume. If the CO allows using different principals, e.g. based on a pod configuration, these individual identities will be applied for the different configured mounts (in this scenario the sub containers have their own individual mounts of the external volumes provided via the SP plugin).
If a root container provides access to his own external volume mount for his child containers by different means the scenario goes out of scope for the SP plugin.